{{cookiecutter.github_project_name}}/package.json

```diff
@@ -90,6 +91,13 @@
     "webpack-cli": "^3.1.2"
   },
   "jupyterlab": {
-    "extension": "lib/plugin"
+    "extension": "lib/plugin",
+    "outputDir": "{{ cookiecutter.python_package_name }}/labextension/",
+    "sharedPackages": {
+      "@jupyter-widgets/base": {
+        "bundled": false,
```

Comment:
Just wanted to say it's amazing this is finally all coming together! Thank you @blink1073 and @jasongrout for all your work here <3

Reply:
I agree, thanks for the amazing work!